Networking/Networking/Remote/ProductsRemote.swift

```diff
@@ -202,11 +202,11 @@ public final class ProductsRemote: Remote, ProductsRemoteProtocol {
     /// - Parameters:
     /// - siteID: Site for which we'll fetch remote products.
     /// - productTypes: A list of product types to be included in the results.
-    /// - pageNumber: Number of page that should be retrieved.
+    /// - pageNumber: Number of pages that should be retrieved.
```

Comment:
nit: I think this parameter is the page index (starting with 1), not the number of pages (total page count)?

Reply:
Yep, you're absolutely right! I was obviously not thinking straight, I'll fix it.